src/components/navigations/app-bar/app-bar.component.js

```diff
@@ -53,9 +86,30 @@ const AppBar = ({ toggleMobileDrawer }) => {
           edge="end"
           aria-haspopup="true"
           color="inherit"
+          onClick={handleClick}
         >
           <AccountCircle />
         </IconButton>
+        <StyledMenu
+          id="customized-menu"
```

Comment:
Give this a more meaningful id, like: `user-action-menu`

Reply:
StyleMenu is no longer available.